src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int COLORS_AMOUNT = Color.values().length;
+    Random random = new Random();
+
+    public String getRandomColor() {
+        Color color = Color.values()[random.nextInt(COLORS_AMOUNT)];
+        return color.toString();
```

Comment:
Use the `name()` method instead of `toString()` when you want to get the String representation of an enum constant. This ensures that you always get the exact name of the enum constant as declared in the enum definition.